spec/mongoid/association/referenced/has_many/enumerable_spec.rb

```diff
@@ -201,6 +201,122 @@
     end
   end
 
+  describe "#empty?" do
+
+    let(:person) do
+      Person.create
```

Comment:
`create!` please, and elsewhere.

Reply:
I'll do it here, but we really should just create a PR which does this across the entire repo. The reason I didn't use `create!` is because I copy-pasted from existing specs.

I can raise the PR once I have CI visibility. It will break many tests.

Reply:
Fixed in this PR.